addons/pos_restaurant/models/pos_config.py

```diff
@@ -63,3 +63,13 @@ def write(self, vals):
         if ('is_order_printer' in vals and vals['is_order_printer'] == False):
             vals['printer_ids'] = [(5, 0, 0)]
         return super(PosConfig, self).write(vals)
+
+    def get_all_table_draft_orders(self):
+        self.ensure_one()
+        self = self.with_context(prefetch_fields=False)
```

Comment:
You don't need this on `self`, but from what I can see, this is intended on `pos.order` to optimize code inside method `get_table_draft_orders()`.  Note that it may also _deoptimize_ possible overrides of the method when used that way!
Such an optimization with `prefetch_fields` should never be global and should local to where it is needed.  Move it where it is needed in `get_table_draft_orders()`.

Reply:
Done